modules/apps/comment/comment-taglib/src/main/resources/META-INF/resources/discussion/js/ViewMessageThread.js

```diff
@@ -12,7 +12,11 @@
  * details.
  */
 
-export default function ViewMessageThread({index, namespace, randomNamespace}) {
+export default function ViewMessageThread({
+	index,
+	portletNamespace: namespace,
```

Comment:
I'm using the `portletNamespace` attr instead of `namespace` to avoid being overwritten here [ComponentTag.java#L229-L233](https://github.com/liferay/liferay-portal/blob/fd14dbe69f730c0a07464e8996c1010942dcb484/modules/apps/frontend-taglib/frontend-taglib/src/main/java/com/liferay/frontend/taglib/servlet/taglib/ComponentTag.java#L229-L233) in the component I rename to a `namespace` to minimize the changes.

We pass the namespace merged with the random namespace and the value isn't the same
![image](https://user-images.githubusercontent.com/67901/178705219-8062c8dd-1a8d-49e8-83f2-720972967cc2.png)
![image](https://user-images.githubusercontent.com/67901/178705275-b6836670-b5fc-404f-807b-9d9e2b78bf9f.png)

Reply:
so if I understood you correctly, `portletNamespace = namespace + randomNamespace`. And, in your component, you are "renaming" this _portletNamespace_ to _namespace_ 

> in the component I rename to a namespace to minimize the changes.

Thanks for your clarification. Although, I would have prefered you to use the portletNamespace variable to avoid confusions, but as you wish.